libs/net/dal/Helpers/FfmpegHelper.cs

```diff
@@ -1,124 +1,152 @@
     {
-        var format = Path.GetExtension(fileName).Replace(".", "");
+        var directory = Path.GetDirectoryName(path) ?? "";
+        var ext = Path.GetExtension(path).Replace(".", "");
         var duration = end - start;
-        var input = directory + "/" + fileName;
-        var output = directory + "/" + prefix + "_" + clipNbr + "." + format;
-        var otherArgs = "-ab 64k -ar 22050 -vol 400";
 
-        return $"ffmpeg -ss {start} -f {format} -t {duration} -i {input} {otherArgs} -y {output}";
+        // Ensures we keep creating new files and never overwrite.
+        var count = Directory.GetFiles(directory, $"{outputName}_*.{ext}").Length;
+        var output = $"{Path.Combine(directory, outputName)}_{count + 1}.{ext}";
+
+        return ($"ffmpeg -ss {start} -t {duration} -i {path} -c:a copy {output}", output);
     }
 
     /// <summary>
-    /// Get the process for the specified clip command.
+    /// Generate the command for the clip process.
     /// </summary>
-    /// <param name="directory"></param>
-    /// <param name="fileName"></param>
-    /// <param name="path"></param>
+    /// <param name="path">Full path to the source file.</param>
     /// <param name="muxFile"></param>
-    /// <param name="format"></param>
-    /// <param name="prefix"></param>
+    /// <param name="prefix">Filename prefix to search for files to join.</param>
     /// <returns></returns>
-    public static System.Diagnostics.Process GetJoinProcess(string directory, string fileName, string path, string muxFile, string format, string prefix)
+    private static (string Cmd, string Output) GenerateJoinCommand(string path, string muxFile, string prefix)
     {
-        var cmd = GenerateJoinCommand(directory, fileName, path, muxFile, format, prefix);
-        var process = new System.Diagnostics.Process();
-        process = InitializeProcess(process, cmd, "join");
+        var ext = Path.GetExtension(path).Replace(".", "");
+        var directory = Path.GetDirectoryName(path) ?? "";
+        var output = Path.Combine(directory, $"{prefix}-final.{ext}");
 
-        return process;
+        return ($"ffmpeg -f concat -safe 0 -i {muxFile} {output}", output);
     }
 
     /// <summary>
-    /// Get the process for the specified clip command.
+    /// Create a muxfile that joins multiple clips into a snippet.
+    /// </summary>
+    /// <param name="path">Full path to the source file.</param>
+    /// <param name="prefix">Filename prefix to search for files to join.</param>
+    /// <returns></returns>
+    private static string GenerateMuxfile(string path, string prefix)
+    {
+        var ext = Path.GetExtension(path).Replace(".", "");
+        var directory = Path.GetDirectoryName(path) ?? "";
+        var clips = Directory.GetFileSystemEntries(directory, prefix + "_*." + ext);
+        // TODO: Two users with same prefix will overwrite each other.
+        var muxfile = Path.Combine(directory, "_tmp", $"{prefix}.txt");
```

Comment:
The file is placed in the current directory to limit the possibility of overwriting another user's file for now.  Need to handle this better in the future.